ansible/environments/mac/group_vars/all

```diff
@@ -4,11 +4,6 @@ whisk_logs_dir: /Users/Shared/wsklogs
 docker_registry: ""
 docker_dns: ""
 
-# The whisk_api_localhost_name is used to configure nginx to permit vanity URLs for web actions.
-# It is also used for the SSL certificate generation. For a local deployment, this is typically
-# a hostname that is resolved on the client, via /etc/hosts for example.
-whisk_api_localhost_name: "openwhisk"
```

Comment:
With this pr, `mac` environment is for Docker for Mac, and Ansible can deploy OW on Mac with configurations in `local` environment.
So I copied `group_vars/all` from local environment.

AFAIK, Ansible can not deploy OW without that option in `docker-machine` env, but in `mac` env, it does not.